### DISCUSSION TITLE
Game Name Ideas

### DISCUSSION THREAD

**Discussion by Mayank0255** [Amegma/Space-Invaders#37] [Ideas]
This discussion is to take inputs from anyone and based on that come up with a new name for Space Invaders as the plan now is to take it away from the Original Atari Space Invaders, by adding more features and scale it up

**Comment by Mayank0255** [comment]
My first suggestion - Galaxy Attack

**Reply by H4ppy-04** [reply]
A couple of ideas 
 * Droid Destroyer
 * Drone Destroyer
 * Galaxy Gateway
 * Star Shooter

Not great at names and I think the one you have here is fine but if you still wanted more a year later. I know it's not too helpful tho but I just saw your post on the pygame discord and thought it was really cool. I wish you luck with development good sir

**Reply by Mayank0255** [reply]
Thanks for the suggestions, will surely consider these
